CONTRIBUTING.md

```diff
@@ -214,11 +214,6 @@ Plus some suggestions, rather than rules:
 * property of a "plain" JSON object that is not an OpenAPI-defined Foo Object -> "property"
 * "attribute" is only used in the XML context and means "XML attribute"
 
-### Field Names and Values in YAML comments
-
-Field names and keywords should be in backticks.
-Values like "Dog" should be double quoted.
-
```

Comment:
This section was added to the style guide in `main` and should never have been added to this outdated state of CONTRIBUTING

Reply:
I saw this change did make it into `main` over in `style-guide.md`. :+1: